CLI.ps1

```diff
@@ -0,0 +1,45 @@
+#Grant super-user privileges
+Start-Process Powershell -Verb runAs
+
+
+#create a directory
+New-Item -ItemType directory -Path C:\DEVOPS-Copy\DevOps-Course-Starter
+
+#Check which Python versions are available 
+#If python is not installed, get the version and store in directory location above
+
+$ispythoninstalled = python --version
+if ($ispythoninstalled) { python --version } else { Invoke-WebRequest -Uri "https://www.python.org/ftp/python/3.7.0/python-3.7.0.exe" -OutFile "C:\DEVOPS-Copy\DevOps-Course-Starter/python-3.7.0.exe" }
+
+
+#command to install Python
+C:\DEVOPS-Copy\DevOps-Course-Starter/python-3.7.0.exe /quiet InstallAllUsers=0 PrependPath=1 Include_test=0
+
+##update the path 
+ENV:PATH="$ENV:PATH;C:\Users\Rajesh\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\Scripts"
```

Comment:
Shouldn't it be `$ENV:PATH="...`?

Also:
- This is very specific to your machine, could you make this general?
- This only updates the PATH environment variable for the current process. If you opened up a new powershell terminal on this machine, it won't know about this. To update your profile so the value is available for all future shells, there's a more complicated command along the lines of:
```
[System.Environment]::SetEnvironmentVariable('ENV_VAR_NAME', 'env var value', [System.EnvironmentVariableTarget]::User)
```
depending on the exact behaviour you want

Reply:
Yes, what you pointed out is correct. For the comment on PATH , I believe this is what I was trying to play with at the time which somehow managed to clear out variables from my settings.